lightning/src/ln/chan_utils.rs

```diff
@@ -212,8 +217,9 @@ impl CounterpartyCommitmentSecrets {
 		Ok(())
 	}
 
-	/// Can only fail if idx is < get_min_seen_secret
-	pub(crate) fn get_secret(&self, idx: u64) -> Option<[u8; 32]> {
+	/// Returns the secret at index `idx`.
+	/// Returns `None` if idx is < [CounterpartyCommitmentSecrets::get_min_seen_secret].
```

Comment:
Use ticks around identifiers.

Reply:
Done (I think it was only `idx` on second line missing?)

Reply:
Also: [\`CounterpartyCommitmentSecrets::get_min_seen_secret\`]

Reply:
Fixed thanks, need to read up more on doc formatting.